app/controllers/EndDateQuestionController.scala

```diff
@@ -44,13 +44,18 @@ class EndDateQuestionController @Inject()(actionsProvider: ActionsProvider,
   def show(taxYear: Int,
            benefitType: BenefitType,
            sessionDataId: UUID): Action[AnyContent] = actionsProvider.endOfYearSessionDataFor(taxYear, benefitType, sessionDataId) { implicit request =>
-    Ok(pageView(EndDateQuestionPage(taxYear, benefitType, request.stateBenefitsUserData, formsProvider.endDateYesNoForm(taxYear))))
+    Ok(pageView(EndDateQuestionPage(
```

Comment:
Can wee make this shorter by doing extracting a variable and adding everything else in one line, e.g.?
val userData = request.stateBenefitsUserData